TAGradingServer/data/tables.sql

```diff
@@ -0,0 +1,520 @@
+--
```

Comment:
Make whatever change you had in this file to ```site/data/tables.sql```. This one is going away and should not be brought back.